File: 02_classes_metodos/ruby/ex01.rb
class Pessoa
  def initialize(nome, idade)
    @nome = nome
    @idade = idade
  end

  def apresentar
    puts "Oi, meu nome é #{@nome} e eu tenho #{@idade} anos."
  end
end


if __FILE__ == $0
  pessoa = Pessoa.new("Deivid", 37)
  pessoa.apresentar
end

File: 02_classes_metodos/python/ex01.py
class Pessoa:
    def __init__(self, nome, idade):
        self.nome = nome
        self.idade = idade
    
    def apresentar(self):
        print(f"Ola, meu nome é {self.nome} e eu tenho {self.idade} anos.")


if __name__ == "__main__":

    pessoa = Pessoa("Deivid", 36)
    pessoa.apresentar()